public/app/plugins/panel/geomap/utils/utils.ts

```diff
@@ -116,3 +116,10 @@ export const getNextLayerName = (panel: GeomapPanel) => {
 
   return `Layer ${Date.now()}`;
 };
+
+export const isUrl = (url: string) => {
```

Comment:
We might want to consider using a built in library function for this so we don't have to own / maintain this 🤔 maybe we can double check in #frontend slack channel if we already have a common way of doing this check?

Reply:
@nmarrs - updated with Gilles' suggestion. I was having some second thoughts this morning about over-complicating the validation.. in the end I don't think we want to validate users' data, but make it clickable if it resemble an url (_http_). If the url is valid or not, might be someone else's problem..

Reply:
Yeah that's a good point. If the image shows up or not is validation itself 😅 before we promote this later we may want to consider having someone in the security team test for vulnerabilities given we are displaying images based on user provided data